mbedtls/src/ecp/mod.rs

```diff
@@ -309,10 +309,6 @@ impl EcPoint {
         Mpi::copy(&self.inner.Y)
     }
 
-    pub fn z(&self) -> Result<Mpi> {
```

Comment:
Why is this being removed?

Reply:
Internally, Jacobian coordinates `(X, Y, Z)` are used to represent an EC point - this is purely for efficiency in addition formulæ. MbedTLS says that every function manipulating a point is expected to accept it and return it with `Z = 1` (or `Z = 0` if it is the infinity point), so that the Jacobian representation is `(x, y, 1)` where `(x, y)` are the Cartesian coordinates. Note that the previous though is only documented and not enforced in any way. Also, Jacobian representation is not unique.